opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/RuntimeEnvironment.java

```diff
@@ -1591,6 +1597,18 @@ public synchronized AuthorizationFramework getAuthorizationFramework() {
         return authFramework;
     }
 
+    /**
+     * Return the analyzer guru.
+     *
+     * @return the analyzer guru instance, never {@code null}.
+     */
+    public synchronized AnalyzerGuru getAnalyzerGuru() {
+        if (analyzerGuru == null) {
+            analyzerGuru = new AnalyzerGuru(getPluginDirectory());
```

Comment:
I wonder if we should just split these 2 plugin directories into analyzers and authorization plugins. This way it is kind of confusing.

Reply:
It's not necessary as it's called plugins directory and you can put all your plugins there be it analyzer or authorization plugin. The application frameworks will load only the plugins they support.

However, if it's desired I can split these.